modules/conversion-center/assets/scss/widgets/contact-buttons.scss

```
@@ -264,4 +259,145 @@
             width: var(--e-contact-buttons-size-large);
         }
 	}
+
+    &__message-bubble {
+        background-color: var(--e-contact-buttons-message-bubble-chat-bg);
+        padding: 20px 20px 20px 40px;
+    }
+
+    &__bubble {
+        background-color: var(--e-contact-buttons-message-bubble-bubble-bg);
+        border-radius: 15px;
+        padding: 20px;
+        position: relative;
+
+        &::after {
+            border-color: transparent var(--e-contact-buttons-message-bubble-bubble-bg) transparent transparent;
+            border-style: solid;
+            border-width: 0 40px 40px 0;
+            content: "";
+            height: 0;
+            position: absolute;
+            left: -20px;
+            top: 0;
+            transform: rotate(0deg);
+            width: 0;
+        }
+    }
+
+    &__message-bubble-name {
+        color: var(--e-contact-buttons-message-bubble-name);
+        font-size: 20px;
+        line-height: 25px;
+        font-weight: 600;
+        margin-bottom: 8px;
+    }
+
+    &__message-bubble-body {
+        color: var(--e-contact-buttons-message-bubble-body);
+        font-size: 20px;
+        line-height: 25px;
+        margin-bottom: 8px;
```

```suggestion
        margin-block-end: 8px;
```
Try to use CSS Logical properties in all the margins/paddings.

---

done!